addons/mail/static/src/components/notification_list/notification_list.scss

```diff
@@ -2,32 +2,10 @@
 // Layout
 // ------------------------------------------------------------------
 
- .o_NotificationList {
-    display: flex;
-    flex-flow: column;
-    overflow: auto;
-
-    &.o-empty {
-        justify-content: center;
```

Comment:
Are you sure you want to remove it ? If not, you can add `justify-content-center` next to `o-empty` in the XML.